CONTRIBUTING.md

```diff
@@ -133,3 +133,18 @@ Use your real name (sorry, no pseudonyms or anonymous contributions).
 
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
+
+### Code of Conduct
+
+CppMicroServices.org welcomes developers with different backgrounds and a broad range of 
+experience. A diverse and inclusive community will create more great ideas, provide more unique 
+perspectives, and produce more outstanding code. Our aim is to make the CppMicroServices community 
+welcoming to everyone.
+
+To provide clarity of what is expected of our members, CppMicroServices has adopted the code 
+of conduct defined by [contributor-covenant.org][contributor-covenant.org]. This document is used 
+across many open  source communities, and we believe it articulates our values well. 
+
+The full text is on [Code of Conduct](./CodeOfConduct.md).
```

Comment:
This sentence sounds awkward to me.
Some alternatives I can think of...

- The full text can be found within our [Code of Conduct](./CodeOfConduct.md).
- The full text is within our [Code of Conduct](./CodeOfConduct.md).
- Please refer to the [Code of Conduct](./CodeOfConduct.md) for further details.